kie-wb-smoke-tests/src/test/java/org/kie/smoke/wb/AbstractWorkbenchIntegrationTest.java

```diff
@@ -23,7 +23,7 @@
 
     static {
         try {
-            deploymentUrl = new URL(System.getProperty("deployable.base.uri"));
+            deploymentUrl = new URL(System.getProperty("deployable.base.uri", "http://localhost:8080/kie-wb/"));
```

Comment:
Oops, this made it in by accident, but I agree with your second suggestion ("wb"). 

Will change it to that.